### PR TITLE
ci: added ping-for-attention command to notify the reviewers

### LINKED ISSUE #211: We need another command for PRs which acts as re-request for review and also reminder for codeowners
We have cases like https://github.com/asyncapi/spec/pull/847#pullrequestreview-1331087602 when PR owner has hard time:
- pinging codeowners that did not yet perform any review on PR
- re-requesting review from people that already did a review, but the native re-request button feature in side bar do not work well

So we have already commands like, `ready-to-merge` or `autoupdate`. We should have more:
- `ping-for-attention`
- `rerequest`

quick proposals ☝🏼 that of course can be changed

---
before we talk about how to parse `codeowners` file, and then get data needed for requesting review, I suggest someone first check GitHub GraphQL or REST APIs as I think one day when I was exploring it, there was some API, probably GitHub Graphql mutation to trigger re-request review.

also maybe there are ready GitHub actions that we could use in workflows and there is no need to call API manually 🤔

### PR DESCRIPTION
**Pull request by Priyansh61**
**Description**
The PR adds the functionality to use `/pfa`or `/ping-for-attention`.
The functionality is achieved by
- Gets the PR data on which we use the command.
- Gets the list of requested_reviewers.
- Checks for reviewers who have made a review.
- Finally pings the reviewrs by adding a comment who have not reviewed.

**Proof of Work**

https://github.com/asyncapi/website/assets/96513964/32352875-0ef7-4e33-8989-c4de00f9ee1e

**Related issue(s)**

Resolves #211

### CODE REVIEW

**Review by derberg — CHANGES_REQUESTED** [review]
hey, thanks for the PR, please have a look at the bot message and failing logs and update the PR title 🙏🏼

**Comment by alequetzalli** [comment]
This will be useful for people like me 😅

File: .github/workflows/help-command.yml [hunk 1]
```diff
@@ -33,7 +33,11 @@ jobs:
         
                     - `/ready-to-merge` or `/rtm` - This comment will trigger automerge of PR in case all required checks are green, approvals in place and do-not-merge label is not added
                     - `/do-not-merge` or `/dnm` - This comment will block automerging even if all conditions are met and ready-to-merge label is added
-                    - `/autoupdate` or `/au` - This comment will add `autoupdate` label to the PR and keeps your PR up-to-date to the target branch's future changes. Unless there is a merge conflict or it is a draft PR.`
+                    - `/autoupdate` or `/au` - This comment will add `autoupdate` label to the PR and keeps your PR up-to-date to the target branch's future changes. Unless there is a merge conflict or it is a draft PR.
+                    - `/ping-for-attention` or `/pfa` or `/ptal` - This comment will add a comment to the PR asking for attention from the reviewrs who have not reviewed the PR yet.
```
Comment (derberg) [inline .github/workflows/help-command.yml:37]: I think we should just go with one way, and `ptal` is the best one as it is known short, just do `ptal` and longer version which probably is something like `please take a look` (probably something to check with urban dictionary)

Comment (Priyansh61) [reply .github/workflows/help-command.yml]: Done

**Review by KhudaDad414 — DISMISSED** [review]
LGTM!

**Review by derberg — COMMENTED** [review]
@Priyansh61 hey, you added to help command info about `rerequest` and `review` but the PR contains only support for `ptal` 🤔

**Comment by derberg** [comment]
unless the plan is to have it here, to not add it in https://github.com/asyncapi/.github/pull/243/files to not face merge conflicts later? so we basically wait for https://github.com/asyncapi/.github/pull/243/ until it is approved and we merge both PRs together?

@KhudaDad414 as you were in review when I was out, did you discuss that these new workflows need to be added to https://github.com/asyncapi/.github/blob/master/.github/workflows/global-replicator.yml#L141?

**Comment by KhudaDad414** [comment]
@derberg my bad, totally forgot that. 🤦

**Comment by derberg** [comment]
@KhudaDad414 since a lot for work is done, but contributor is not around, maybe you can just commit directly to his for what is missing and we can merge?

**Review by KhudaDad414 — DISMISSED** [review]
@derberg everything looks good from my side. 👍

**Review by derberg — DISMISSED** [review]
🚀

**Review by KhudaDad414 — APPROVED** [review]

**Pull request merged**